pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -10,17 +10,20 @@ let
     src = fetchFromGitHub {
       owner = "supermerill";
       repo = "SuperSlicer";
-      sha256 = "sha256-Gg+LT1YKyUGNJE9XvWE1LSlIQ6Vq5GfVBTUw/A7Qx7E=";
+      sha256 = "sha256-W+0lbwWoaqaKqUWyFdETi6F/5dMfad+aUhNohG7O+Mg=";
       rev = version;
       fetchSubmodules = true;
     };
 
     # We don't need PS overrides anymore, and gcode-viewer is embedded in the binary.
     postInstall = null;
+    dontStrip = true;
+    separateDebugInfo = true;
 
     # See https://github.com/supermerill/SuperSlicer/issues/432
     cmakeFlags = super.cmakeFlags ++ [
       "-DSLIC3R_BUILD_TESTS=0"
+      "-DCMAKE_BUILD_TYPE=Debug"
```

Comment:
building without optimizations is still problematic. you should still have reasonable backtraces with separateDebugInfo even with a release build.

Reply:
I've tried it, and there were no line info left.

Reply:
The crux of the problem is compiling with `-g` but not with `-O0`. Either you can find a cmake flag for this, or maybe you can try
```
NIX_CFLAGS_COMPILE="-g";
```